code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -214,6 +214,13 @@
 			N.adjustBruteLoss(-1)
 			N.adjustFireLoss(-1)
 
+/obj/item/weapon/reagent_containers/food/snacks/cucumberslice
+	name = "Cucumber Slice"
+	desc = "A slice of cucumber."
+	icon_state = "cucumberslice"
+	list_reagents = list("nutriment" = 0.01)
```

Comment:
In this case, each slice gets roughly 1/5 of the reagents in the original cucumber, so if the cucumber had 35 units total, each slice would get 7 of it (2 water, 4 plantmatter, and 1 vitamin in this case)